### ISSUE TITLE
fr:number/fr:currency label doesn't focus field

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#1574]
With #1553 we can associate `@for` with an internal control. But this doesn't work for fr:number/fr:currency as they use an HTML input control.

**Comment by avernet** [comment]
Closing: clicking on the label correctly sets the focus on the field, and both `currency.xbl` and `number.xbl` had a `xxbl:label-for` for a while now.
